fvtr/speccpu2017/speccpu2017.exp

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/expect
+# Copyright 2018 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check if AT can compile and run some SPEC CPU 2017 benchmarks
+# The SPEC CPU 2017 must be installed and it's environment set, as explained at
+# http://www.spec.org/cpu2017/Docs/install-guide-unix.html#s6
+
+source ./shared.exp
+
+if { [array names env -exact SPEC] == "" } {
```

Comment:
How do you distinguish that SPEC CPU**2017** is installed (and active (i.e.: `shrc` was executed)) rather than SPEC CPU**2006**?  If I recall correctly, both set the `SPEC` environment variable.

Reply:
@gftg85: You are correct.
The only way I can think right now is to check the name of the directory in the SPEC environment variable, but this way we need to force spec to be installed with a given name.
I modified the patch to check for cpu2017 or cpu2006.
Do you have any suggestions?